### ISSUE TITLE
Error with Container and Text

### ISSUE THREAD

**Issue by Jeremy-Knudsen** [yuanqing/create-figma-plugin#36]
@yuanqing When I create a new plugin from scratch using the `figma-hello-world` template, and then I follow the steps to add `ui.tsx` with the below code, I get the following errors for `Container` and `Text`:

**Container error**
> Type '{ space: "medium"; }' is not assignable to type 'Props<HTMLDivElement, ContainerProps>'.
>   Property 'children' is missing in type '{ space: "medium"; }' but required in type 'ContainerProps'.ts(2322)
> container.d.ts(4, 5): 'children' is declared here.
> 

... and the following error for `Text`:

**Text error**
> Type '{}' is not assignable to type 'Props<HTMLDivElement, TextProps>'.
>   Property 'children' is missing in type '{}' but required in type 'TextProps'.ts(2322)
> text.d.ts(4, 5): 'children' is declared here.
> 

**Code from https://yuanqing.github.io/create-figma-plugin/#configuration**

`import { render, Container, Text, VerticalSpace } from '@create-figma-plugin/ui'`
`import { h } from 'preact'`

`function Plugin (props: { greeting: string }) {`
`  return (`
`    <Container space='medium'>`
`      <VerticalSpace space='medium' />`
`      <Text>{props.greeting}</Text>`
`      <VerticalSpace space='medium' />`
`    </Container>`
`  )`
`}`

`export default render(Plugin)`

**Comment by Jeremy-Knudsen** [comment]
Interestingly, this will still build successfully without errors even though I see the wavy underlines in VS Code
![image](https://user-images.githubusercontent.com/11559173/119397990-c5cdb180-bc8b-11eb-8e22-9044f4555db7.png)

![image](https://user-images.githubusercontent.com/11559173/119398048-dc740880-bc8b-11eb-9bbc-b1d2cbe6ef9b.png)

![image](https://user-images.githubusercontent.com/11559173/119398100-edbd1500-bc8b-11eb-90b0-f4e23d81398d.png)

**Comment by yuanqing** [comment]
@Jeremy-Knudsen 

Thanks for reporting this! Am able to repro this. The fix is to update the **`"include"`** key in the `tsconfig.json` to also include `"src/**/*.tsx"`:

```
{
  "extends": "@create-figma-plugin/tsconfig",
  "compilerOptions": {
    "typeRoots": ["node_modules/@figma", "node_modules/@types"]
  },
  "include": ["src/**/*.ts", "src/**/*.tsx"]
}
```

**Comment by Jeremy-Knudsen** [comment]
That resolves the bug I'm seeing! Thank you @yuanqing ... I love this repo and the work you are doing here!

**Comment by Jeremy-Knudsen** [comment]
Before closing this issue, @yuanqing I think it would be good to include a topic in the documentation about the `tsconfig.json` file in more detail.

**Comment by yuanqing** [comment]
Bug should be fixed as of `1.0.2`
